iocore/net/SSLUtils.cc

```diff
@@ -141,8 +142,9 @@ static int ssl_vc_index = -1;
 static ink_mutex *mutex_buf      = nullptr;
 static bool open_ssl_initialized = false;
 
-RecRawStatBlock *ssl_rsb                   = nullptr;
-static InkHashTable *ssl_cipher_name_table = nullptr;
+RecRawStatBlock *ssl_rsb = nullptr;
+typedef std::unordered_map<const char *, void *> cipherTable;
```

Comment:
Why not make the value type `intptr_t` or even `int` to avoid the cast to `void*`? The latter is purely an artifact of TCL hash table.